frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -736,9 +736,26 @@ impl<T: Trait> Sign<BlockNumber<T>> for Module<T> {
 		message: impl AsRef<[u8]>,
 		signer: Self::Signer,
 	) -> bool {
-		if let Ok(public_key) =
-			crypto::secp256k1_ecdsa_recover(signature, &hashing::blake2_256(message.as_ref()))
-		{
+		fn eth_signable_message(message: &[u8]) -> Vec<u8> {
+			let mut l = message.len();
+			let mut rev = Vec::new();
+
+			while l > 0 {
+				rev.push(b'0' + (l % 10) as u8);
+				l /= 10;
+			}
```

Comment:
What's this?

Reply:
May be related to https://docs.ethers.io/ethers.js/v5-beta/cookbook-signing.html

Reply:
> What's this?

Same things, in darwinia-claims.